lib/debugger.coffee

```diff
@@ -31,11 +35,15 @@ class ProcessManager extends EventEmitter
         appArgs = @atom.config.get('node-debugger.appArgs')
         port = @atom.config.get('node-debugger.debugPort')
         env = @parseEnv @atom.config.get('node-debugger.env')
+        defaultFile = @atom.project.getPaths()[0] + '/' + @atom.config.get('node-debugger.defaultFile')
```

Comment:
well that's...  meh
i'm going to dig around the `@atom` object.  I think i have an idea.